webprojects/SUMO.md

```diff
@@ -0,0 +1,77 @@
+# Support.mozilla.org(SUMO): how to localize and test
+
+SUMO, short for **SU**pport.**MO**zilla.org, is a knowledge base site, providing support to Mozilla product users. More than half of all Firefox users speak a language other than English. Mozilla depends on contributors to make support available to people around the globe.
+
+SUMO consists of two parts:
+* User interface (or UI), such as navigation and buttons.
+* Support articles, either use to troubleshoot issue or as knowledge base for Mozilla products.
+
+This document only covers the UI portion localizable in Pontoon.
+
+## Localizing the strings
+
+There are two active components in Pontoon that make up the [SUMO product](https://pontoon.mozilla.org/sumo):
+
+* **LC_MESSAGES/djangojs.po** contains strings for the JS bits of the platform.
+* **LC_MESSAGES/django.po** contains the majority of strings in the SUMO product.
+
+SUMO is made up by several file types. In order to minimize revision and get translation right the first time, the key is to get the context right. Look for clues or ask others for help:
+* [String ID](https://github.com/mozilla-l10n/sumo-l10n/blob/master/en_US/LC_MESSAGES/django.po#L265) shows where the strings is used.
```

Comment:
This is more confusing than before. You're talking about Pontoon and then pointing people to a GitHub repository they might not even know about…

There's no such thing as a String ID in .po files, what you're pointing to it's a comment.

Let's drop this line.